streams/src/main/java/org/apache/kafka/streams/errors/LogAndContinueExceptionHandler.java

```diff
@@ -17,47 +17,27 @@
 package org.apache.kafka.streams.errors;
 
 import org.apache.kafka.clients.consumer.ConsumerRecord;
-import org.apache.kafka.streams.processor.ProcessorContext;
+import org.apache.kafka.streams.StreamsConfig;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.util.Map;
 
+import static org.apache.kafka.streams.errors.internals.ExceptionHandlerUtils.maybeBuildDeadLetterQueueRecords;
+
 /**
  * Deserialization handler that logs a deserialization exception and then
  * signals the processing pipeline to continue processing more records.
  */
 public class LogAndContinueExceptionHandler implements DeserializationExceptionHandler {
     private static final Logger log = LoggerFactory.getLogger(LogAndContinueExceptionHandler.class);
+    private String deadLetterQueueTopic = null;
 
-    /**
-     * @deprecated Since 3.9. Use {@link #handle(ErrorHandlerContext, ConsumerRecord, Exception)} instead.
-     */
-    @SuppressWarnings("deprecation")
-    @Deprecated
     @Override
-    public DeserializationHandlerResponse handle(final ProcessorContext context,
```

Comment:
For my understanding - Why can we just remove this?

Don't we have to keep it around for compatibility?

Reply:
Ah, I was asking because this PR seems to remove it. But I realize now that there is still a default implementation in the interface that throws a `throw new UnsupportedOperationException();`.